tests/index.spec.js

```diff
@@ -23,14 +23,6 @@ describe('rc-switch', () => {
     expect(switcher.state().checked).toBe(false);
   });
 
-  it('should toggle upon space and enter key', () => {
-    expect(switcher.state().checked).toBe(false);
-    switcher.simulate('keydown', { keyCode: 32 });
-    expect(switcher.state().checked).toBe(true);
-    switcher.simulate('keydown', { keyCode: 13 });
-    expect(switcher.state().checked).toBe(false);
-  });
```

Comment:
Why remove this test case?

Reply:
enzyme doesn't fire onClick on such keypresses, but the browsers will call `onClick` because it's button
Plus, if we'll try to test such behavior we will be actually testing a11y specification, which i think we shouldn't do